cwd/editor.js

```diff
@@ -1,3 +1,60 @@
+const update = (btn) => {
+  const viewNum = btn.id.substring(8, 9);
+  const gaugeNum = btn.id.substring(10, 11);
+  const msgNum = btn.id.substring(12, );
```

Comment:
Is the empty second argument purposeful?  I would probably remove the comma if so.

Reply:
An alternate way of doing this that might make it less dependent on magic numbers could be to add additional attributes to the `btn` HTML.  For instance,
`<element id=<viewNum>-<gaugeNum>-<msgNum> viewNum=<viewNum> gaugeNum=<gaugeNum> msgNum=<msgNum>>`

Reply:
This leaves the `id`

Reply:
This is not necessary now that we are using React.